trimmer.rb

```diff
@@ -0,0 +1,8 @@
+require 'decorator'
+class TrimmerDecoratorr < Decorator
+  def correct_name
+    return unless @nameable.correct_name.size <= 10
+
+    @nameable.correct_name.strip
+  end
+end
```

Comment:
- [ ] This method doesn't work as intended still. You can use something like the following to ensure that it's working properly.
```ruby
@nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
```